robots/pr-creator/main.go

```diff
@@ -100,7 +102,14 @@ func main() {
 		logrus.WithError(err).Fatal("Failed to create github client")
 	}
 
-	n, err := updater.EnsurePR(o.org, o.repo, o.title, o.body, o.source, o.branch, o.headBranch, o.allowMods, gc)
+	var queryTokensString string
+	// Prioritize using headBranch as it is less flakey
+	if o.headBranch != "" {
```

Comment:
Should there be a check added to validate they are not both set? Or mention one has priority in the options?

Reply:
Added to the options that headBranch overrides match-title.